packages/tasks/src/tasks/feature-extraction/inference.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+
+/**
+ * Inputs for Feature Extraction inference
+ */
+export interface FeatureExtractionInput {
+	/**
+	 * One or several texts to get the features of
+	 */
+	inputs: string[] | string;
```

Comment:
We need to decide if we want to support batched inputs or not. Community API does not support batched inputs, Inference API does support them.

Reply:
Maybe let's keep it simple for those types and remove batching then?